client/js/components/user/DpUserFormFields.vue

```diff
@@ -329,6 +338,27 @@ export default {
       return new Promise((resolve, reject) => resolve(true))
     },
 
+    fetchOrgaById (orgaId) {
+      const url = Routing.generate('dplan_api_orga_get', { id: orgaId })
+
+      return dpApi.get(url, { include: ['allowedRoles', 'departments'].join() })
+    },
+
+    /**
+     *  Handle cases in which allowedRoles are missing in the orga relationships (after user update action)
+     */
+    getOrgaAllowedRoles (orgaId) {
+      let allowedRoles = this.rolesInRelationshipFormat
+      this.fetchOrgaById(orgaId).then((orga) => {
```

Comment:
nitpick: please add a blankline between the var declaration and the method (logic) block

Reply:
done here https://github.com/demos-europe/demosplan-core/pull/750/commits/51d180a277b07fb9a3561b0093e0500bb086aee1